orc8r/cloud/deploy/bare-metal-ansible/roles/magma/orc8r/templates/orc8r-values.yaml.j2

```diff
@@ -0,0 +1,151 @@
+controller:
```

Comment:
If this is same or similar to existing values, I will prefer not duplicating these.

Reply:
I want to refactor orc8r chart so we can make this values file 20 lines instead of 151. This is a rewrite of what we have for Terraform currently